### ISSUE TITLE
Death Animation in Revive System

### ISSUE THREAD

**Issue by simon84** [drakelinglabs/projectrealityarma3#91]
Since enabling ragdoll as the default death animation is hard to get working in a network performant way, we should consider using a death animation like the ace medical system is using. They have a helper function, which select the right animation for the current animation state of a unit `[bob] call ace_common_fnc_ getDeathAnim`. In my opinion, it is a good workaround.

**Comment by simon84** [comment]
Ok, joko explaned me the shitty animation framework of arma and why it is not working without config modification. :(

That means ragdoll is the only option to get a death animation.

**Comment by simon84** [comment]
Ok, after testing with ragdoll physics in arma, joko and I explored, that damage caused by vehicle collision does not trigger the handleDamage event. This leads to problems with our current medical system, which does not allow vanilla damage. We came to the conclusion, that we should handle the unconscious state in the same way like the vanilla revive system (unit gets killed, new unit after reviving). This also solves the problem of ragdolling by the way.
